stix2/v20/observables.py

```diff
@@ -720,11 +720,11 @@ class WindowsRegistryKey(_Observable):
         ('modified', TimestampProperty()),
         ('creator_user_ref', ObjectReferenceProperty(valid_types='user-account')),
         ('number_of_subkeys', IntegerProperty()),
-        ('extensions', ExtensionsProperty(spec_version="2.0", enclosing_type=_type)),
+        ('extensions', ExtensionsProperty(spec_version='2.0')),
     ])
 
 
-class X509V3ExtenstionsType(_STIXBase20):
+class X509V3ExtensionsType(_STIXBase20):
```

Comment:
This typo fix is great! But could we pull it out into a separate PR? In my mind it's not directly related to the rest of the Extensions support.